wiki/Shortcut_Key_Reference/en.md

```diff
@@ -1,179 +1,202 @@
--   **Spacebar**: Skip opening cut-scene.
--   **Tab**: Toggle scoreboard
--   **Shift+Tab**: Toggle in-game interface
--   **F3/F4**: Adjust speed for osu!mania.
--   **Ctrl+R or \`**: Quick retry the current map (autoskips the cut-scene)
-
-Edit Mode
----------
-
-### Keyboard Shortcuts
-
-#### General
-
-![How it would look like when you press H (Help) key](Edit_Help.jpg "How it would look like when you press H <Help> key")
-
--   **H**: Help (Contains all the shortcuts shown below)
-
-**Music**
-
--   **Z**: Stop
--   **X**: Play from start
--   **C or Spacebar**: Pause/Resume
--   **Left and Right arrow keys**: Skip through the song. Hold shift for 4x speed
-
-**Bookmarks commands**
-
--   **Ctrl+B**: Add bookmark at current location.
--   **Ctrl+Shift+B**: Remove bookmark at current location.
--   **Ctrl+Left Arrow**: Jump to previous bookmark.
--   **Ctrl+Right Arrow**: Jump to next bookmark.
-
-**Save and Reset**
-
--   **Ctrl+L**: Revert to saved
--   **Ctrl+S**: Quick save
--   **Ctrl+N**: Remove all notes (but keep timing/song setup)
-
-**Mode switch**
-
--   **F1**: Compose mode
--   **F2**: Design mode (only basics available at the moment)
--   **F3**: Timing mode
--   **F4**: Song setup
+# Shortcut Key Reference
+
+There are many shortcuts in osu! that can be utilized while in-game or while idle at any screen.
+They are listed out with their functions below.
+
+**Note:** some of these keys can be changed in the [Options](/wiki/Options/)!
+However, the list below presents the default keys.
+
+## Game-Wide
+
+These shortcuts work anywhere:
+
+- `Arrow Down`: Decrease volume.
+- `Arrow Up`: Increase volume.
+- `Alt` + `F4`: If playing/editing a beatmap, exit to song select; otherwise, this will close the osu!client.
+- `Ctrl` + `Alt` + `Shift` + `S`: Reload current skin (may say that it is in queue)
+- `Ctrl` + `F11`: Toggle trame times
+- `Ctrl` + `O`: Open options menu (does not work while playing in-game)
+- `F7`: Toggle FPS limiting mode
+- `F8`: Toggle chat window
+- `F9`: Toggle extended chat window
+- `F10`: Toggle all mouse buttons (except for mouse wheel)
+- `F12`: Take a screenshot (hold `Shift` to upload instantly to the server for sharing.)
+- `Insert`: Minimize the osu!client into the notification area.
+- `Shift` + `F1`: Open [osu!status](https://twitter.com/osustatus "osu!status on twitter") (when connection error appears)
+
+## Main Menu
+
+- `Alt` + `Enter`: Toggle fullscreen mode
+- `Esc`: Go back one menu
+- `P`: Play mode
+- `P`, then `P`: Solo
+- `P`, then `M`: [Multi](/wiki/Multi)
+- `E`: Edit mode
+- `Z` or `F1`: Previous song (jukebox)
+- `X` or `F2`: Play from start (jukebox)
+- `C`: Toggle pause (jukebox)
+- `F4`: Stop playing and reset duration to 0 (jukebox)
+- `V` or `F5`: Next song (jukebox)
+- `J` or `F6`: Jump To (jukebox)
+- `R`: Random song (jukebox)
+- `D`: osu!direct ([osu!supporters](/wiki/osu!supporter) only)
+- `O`: Options
+- `Q` or `Esc`, then `Esc`: Quit (displays a prompt)
```

Comment:
`1` can be used to choose `Exit` on the prompt too.




it might actually be 2 idk because no access to the game